lib/data_exporter.rb

```diff
@@ -61,4 +65,33 @@ def export_csv(subscriber_lists, at: nil)
       end
     end
   end
+
+  def travel_advice_csv(subscriber_lists, at: nil)
+    CSV($stdout, headers: %i(title immediately daily weekly), write_headers: true) do |csv|
+      subscriber_lists.find_each do |subscriber_list|
+        csv << present_travel_advice_report(subscriber_list, at: at)
+      end
+    end
+  end
+
+  def present_travel_advice_report(subscriber_list, at: nil)
+    { "title" => subscriber_list.title.delete(",") }.merge(subscriber_list_frequency_count(subscriber_list, at)).symbolize_keys
+  end
+
+  def subscriber_list_frequency_count(subscriber_list, at)
+    Subscription
+      .where(subscriber_list: subscriber_list)
+      .where("created_at < ?", at)
+      .where("ended_at IS NULL OR ended_at >= ?", at)
+      .group(:frequency)
+      .count
+  end
+
+  def travel_advice_subscriber_lists
+    SubscriberList.where(slug: travel_advice_slugs)
```

Comment:
It would probably be better to query against the subscriber lists which are linked to a `country`. For example:

```rb
SubscriberList.where("links->'countries' IS NOT NULL")
```

That way we don't need to keep the countries list up to date.

Reply:
Great idea, thanks @thomasleese !